translations/nl/LC_MESSAGES/messages.po

```diff
@@ -67,9 +67,8 @@ msgstr "Het lijkt erop dat je vergeten bent om een commando te gebruiken op rege
 msgid "Missing Inner Command"
 msgstr "Het lijkt erop dat je vergeten bent om het {command} commando op regel {line_number} in te vullen."
 
-#, fuzzy
 msgid "Misspelled At Command"
-msgstr "It looks like you might have misspelled the {command} command, instead you wrote {invalid_argument} in line {line_number}."
+msgstr "Je hebt {command} verkeerd geschreven, jij schreef {invalid_argument} op regel {line_number}."
```

Comment:
It seems like the change I did to the plug-in did work properly. I'm afraid we'll have to run the --no-wrap for es and id ourselves, but hopefully in the future we won't have as many changes!

Reply:
Awesome!! Feel free to approve this so we can than do another unwrapping one (and update the Wiki, again?)

Reply:
Wiki updated!